lib/main.js

```diff
@@ -100,6 +100,7 @@ class PythonLanguageClient extends AutoLanguageClient {
               "Make sure to install `pyls` 0.19 or newer by running:\n" +
               "```\n" +
               `${python} -m pip install 'python-language-server[all]'\n` +
+              `${python} -m pip install git+git+https://github.com/tomv564/pyls-mypy.git\n` +
```

Comment:
Is there a reason why we need to install the package from GitHub instead of from the official PyPI release?
```suggestion
              `${python} -m pip install pyls-mypy\n` +
```

Reply:
the reason, I've gone for pip install from github is: https://github.com/tomv564/pyls-mypy/issues/44 (https://github.com/tomv564/pyls-mypy/pull/38 is not yet included in the pip install and you'll get import errors using pyls-mypy.)

Reply:
<img width="586" alt="pyls-mypy-import-error" src="https://user-images.githubusercontent.com/33448151/101158485-17c24280-362c-11eb-9ed2-a285d69731d4.png">